src/diffusers/image_processor.py

```diff
@@ -400,8 +400,7 @@ def binarize(self, image: PIL.Image.Image) -> PIL.Image.Image:
             `PIL.Image.Image`:
                 The binarized image. Values less than 0.5 are set to 0, values greater than 0.5 are set to 1.
         """
-        image[image < 0.5] = 0
-        image[image >= 0.5] = 1
+        image.point(lambda p: 0 if p < 0.5 else 1, "1")
```

Comment:
The code above is more readable and `image.point()` is not extremely widely used unlike the other PIL methods that we use. 

@yiyixuxu WDYT?

Reply:
Ok, that's fine.